templates/emails/subscription.canceled.hbs

```diff
@@ -12,7 +12,10 @@ Subject: Contribution cancelled to {{{collective.name}}}
 
 <p>Your recurring contribution to {{collective.name}} (cced) for {{currency subscription.amount currency=subscription.currency}}/{{subscription.interval}} has been cancelled.</p>
 
-<p>We are sad to see you go 😞. If you have any feedback, we want to hear about it. Just hit 'reply all'.</p>
+{{#if reason}}
+  <p>You provided the following reason for cancelling:</p>
+  <p>{{reason}}</p>
+{{/if}}
```

Comment:
I edited this to:
- Move reason up (it was previously appearing after the closing sentence - `Warmly - xxx`.
- Removed `If you have any feedback, we want to hear about it. Just hit 'reply all'`, this email now uses a no-reply